qiita_pet/handlers/preprocessing_handlers.py

```diff
@@ -17,46 +15,13 @@ def post(self):
         study_id = int(self.get_argument('study_id'))
         prep_template_id = int(self.get_argument('prep_template_id'))
         raw_data = RawData(PrepTemplate(prep_template_id).raw_data)
+        param_id = int(self.get_argument('preprocessing_parameters_id'))
 
         # Get the preprocessing parameters
         if raw_data.filetype == 'FASTQ':
-            form_data = PreprocessIlluminaParametersForm()
```

Comment:
I'm glad to see much of this hard coding removed, but am confused about the removal of WTForms?